hypothesisfuzzer/fuzzer.py

```diff
@@ -71,6 +75,20 @@ def get_commit_hash():
     })
 
 
+@app.errorhandler(500)
+def private_repo():
```

Comment:
can you just add _error to the end of these methods?

makes them a little clearer I think

Reply:
whoops didnt mean to approve, meant to request changes my bad.